### DISCUSSION TITLE
[Art competition] Kasiski's Method

### DISCUSSION THREAD

**Discussion by Re-st** [prosyslab-classroom/cs348-information-security#396] [Art competition]
![DALL·E 2023-05-14 00 53 36 - a fox at the hill, looking at the sunset, waiting for le petit prince to come back](https://github.com/prosyslab-classroom/cs348-information-security/assets/42648067/b698484a-e227-42bc-9f24-ea0336c38436)

## Short Description
For me, [_Kasiski's Method_] means [_a fox at the hill, looking at the sunset, waiting for le petit prince to come back_]".

## Long Description (up to 3 sentences)
Like the fox waiting for the prince to come back, the method checks if the substring is repeated in ciphertext. It will check every distance with anticipation (기대), but it will anticipate even more at the distance multiple of 24 since 24 has many divisors, and so it's more likely the repetance is due to the repetance of plaintext, not by coincidence. Like the fox anticipates the prince to come back at same hour of the day.

**Comment by KihongHeo** [comment]
Very nice!
